app/pb_kits/playbook/pb_distribution_bar/_distribution_bar.scss

```diff
@@ -0,0 +1,29 @@
+@import "../tokens/colors";
+
+[class^=pb_distribution_bar] {
```

Comment:
Currently this needs to be this complex. Playbook kits generate a single, dynamic class based on props.  For example the complete class for this kit could be `pb_distribution_bar_sm`.  `.pb_distribution_bar` would not apply the styles to the element in this case

Reply:
It took me awhile to work through this, but this is what I am understanding these rules to try to generate.

```
[class^=pb_distribution_bar] {
  // common rules for pb distribution bar

  &[class*=_sm] {
    // specific pb distribution bar small rules
  }
  &[class*=_lg] {
    // specific pb distribution bar large rules
  }
}
```
To avoid writing out

```
.pb_distribution_bar_sm {
   // specific pb distribution bar small rules
  // common rules for pb distribution bar
}

.pb_distribution_bar_lg {
  // specific pb distribution bar large rules
  // common rules for pb distribution bar
}
```

Have we considered a mixin to allow the two rule to share the common rules?

I find this code very difficult to read and understand as it is written. I also have concerns with the grep-ability of these kids of rules. Have we considered the performance of rules written like this, I would be suspicious of the rules performance.